tribe-ext-calendar-widget-areas/index.php

```diff
@@ -44,118 +41,181 @@ class Tribe__Extension__Calendar_Widget_Areas extends Tribe__Extension {
 	protected function construct() {
 		// Each plugin required by this extension
 		$this->add_required_plugin( 'Tribe__Events__Main' );
```

Comment:
Don't forget to add `$this->set_version()` here as well. 

https://github.com/moderntribe/products-extensions#updating-an-extension